boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
@miri64 Sorry that I didn't find time to update this PR tonight, I consumed all of my time on another big PR (Lw-MAC), will deal with this tomorrow.